src/gpu/intel/utils.hpp

```diff
@@ -25,7 +25,7 @@
     VCHECK(common, create, check, runtime, stat, msg, ##__VA_ARGS__);
 
 // Uncomment this when aborting on ir_assert is desired:
-// #define GPU_ABORT_ON_ERROR
+#define GPU_ABORT_ON_ERROR
```

Comment:
Leaving a comment to not forget to revert this change back.

Reply:
Thanks, I was under the belief that this was already reverted this, so it was definitely forgotten.